types/trigger.d.ts

```diff
@@ -26,12 +26,12 @@ export type LocaleText = LocaleObject<string>;
 
 export type ZoneId = number | null;
 
+export type OutputStrings = { [outputKey: string]: LocaleText };
+
 // TODO: is it awkward to have Outputs the static class and Output the unrelated type?
 // This type corresponds to TriggerOutputProxy.
 export type Output = {
-  responseOutputStrings: {
-    [outputKey: string]: LocaleText;
-  };
+  responseOutputStrings: OutputStrings | string;
```

Comment:
```suggestion
  responseOutputStrings: OutputStrings;
```

If the above is true, then this doesn't need `| string`.